config/configuration.yaml

```diff
@@ -39,12 +39,45 @@ modbus_mapping:
       scale: 1.0 # Fill this in
       address: [2] # Fill this in with actual Modbus register address for evgBatteryTargetSOCPercent
 
+    - name: "testRegister"
```

Comment:
Not sure if the "Fill this in" comments on every line are all that helpful. Could we just have a single comment at the top of the `modbus_mapping` section saying something like "Replace with settings as appropriate for your site" ? I will be adding some content to the README describing how to work with the config file so we don't need line-by-line instructions

Reply:
similarly to what I said previously, combining the default config file with the integration tests one is something I'd rather avoid. Let's create a separate config file for testing. You could re-use the one you are using for unit tests instead for example.